src/presentation/pages/SignUp/index.tsx

```diff
@@ -115,13 +135,39 @@ const SignUp: React.FC<SignUpProps> = ({
               type="email"
               name="email"
             />
-            <Input
+            <Input className={Styles.signUpPassword}
               autoComplete="off"
               title="Enter your password"
               type="password"
               name="password"
               minLength={4}
             />
+            {(strength=="Strong") ? (
```

Comment:
you can create a generic logic to receive the  strength and render the strength. In this eay, you repeat yourself a lot of times. And compare with 3 === due to compare value and type